ga4gh/datamodel/ontologies.py

```diff
@@ -5,25 +5,38 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
-import fnmatch
-import os
 import ga4gh.protocol as protocol
 
 
-class FileSystemOntology(object):
+class Ontology(object):
```

Comment:
Ah, I get it. What about ``OntologyTermMap`` then? This seems to capture what it is we're using fairly well --- we import an OntologyTermMap into the server to allow us to map from term names to full ontology term IDs?

I'll fix this in a follow-up PR (either way) and make sure that it's all consistent.